### DISCUSSION TITLE
Include tags in Lunr.js offline search

### DISCUSSION THREAD

**Discussion by killermoehre** [google/docsy#1260] [Q&A]
Hi,

was anybody able to include tags and labels in the lunr.js offline search? Something you can access with `tag:<tag_name>` in the search bar?

**Comment by blonewolfs** [comment]
Tags seem to be added to lunr.js automatically. Putting the "tag:" prefix on the search wouldn't find anything for me but, if I just typed in the tag word it would show up like any other search word.

So, on my site if I search for "tag:gpg" it returns nothing but, if I search for "gpg" it returns any docs with gpg (case-insensitive) in the title, tag field, or document body.

I don't currently use anything but tags in my docs but, I assume any other document types like labels or categories would also be included in the search results.
